apis/r/tests/testthat/test-shape.R

```diff
@@ -4,17 +4,20 @@ test_that("SOMADataFrame shape", {
   asch <- create_arrow_schema()
 
   index_column_name_choices = list(
-    "soma_joinid",
-    c("soma_joinid", "foo"),
-    c("soma_joinid", "baz"),
-    c("baz", "foo")
+    one = "soma_joinid",
+    two = c("soma_joinid", "foo"),
+    three = c("soma_joinid", "baz"),
+    four = c("baz", "foo")
```

Comment:
These test modifications can be simplified:
 - the names of the list aren't being used for anything essential
```suggestion
    "soma_joinid",
    c("soma_joinid", "foo"),
    c("soma_joinid", "baz"),
    c("baz", "foo")
```